app/command_autocomplete.go

```diff
@@ -251,3 +259,23 @@ func parseListItems(items []model.AutocompleteListItem, parsed, toBeParsed strin
 	}
 	return true, parsed + toBeParsed, "", suggestions
 }
+
+func parseBoolArgument(arg *model.AutocompleteArg, parsed, toBeParsed string) (found bool, alreadyParsed string, yetToBeParsed string, suggestion model.AutocompleteSuggestion) {
+	in := strings.TrimPrefix(toBeParsed, " ")
+	a := arg.Data.(*model.AutocompleteBoolArg)
+	if in == "" { //The user has not started typing the argument.
+		return true, parsed + toBeParsed, "", model.AutocompleteSuggestion{Complete: parsed + toBeParsed, Suggestion: "", Hint: a.Hint, Description: arg.HelpText}
```

Comment:
We should probably return `true` and `false` as suggestions here

Reply:
yes sure i will update that